site/pages/theme-topic-en.hbs

```diff
@@ -0,0 +1,107 @@
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+			</div>
+		</section>
+		<div class="pull-right col-xs-12 col-md-4">
+			{{>more-info}}
+		</div>
+	</div>
+</div>
+<div class="container">
+	<section class="whtwedo">
```

Comment:
That CSS name was there before? We should consider to use a name with the prefix "gc-"

Reply:
If this don't apply any style at this pattern, remove the CSS class completly